windowsBase/windowsBaseInit.ps1

```diff
@@ -4,7 +4,7 @@ $NODE_ARCHITECTURE = "x86_64"
 $NODE_OPERATING_SYSTEM = "WindowsServer_2016"
 $SHIPPABLE_RELEASE_VERSION = "master"
 $EXEC_IMAGE = "drydock/w16reqproc:master"
-$REQKICK_DIR = "$env:TEMP/reqKick"
+$REQKICK_DIR = "$env:USERPROFILE/reqKick"
```

Comment:
I guess this will change along with https://github.com/Shippable/node/pull/372/files#r173360786